tests/test_functional.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+import socket
+from unittest import mock, TestCase
+from urllib.parse import urlparse
+
+from nose.plugins.attrib import attr
+
+from mtp_transaction_uploader import upload, settings
+
+logger = logging.getLogger('mtp')
+
+
+@attr('functional')
+class FileUploadFunctionalTestCase(TestCase):
+
+    def setUp(self):
+        self.load_test_data()
```

Comment:
in fact, to not duplicate the test data reloading method, could this be `FunctionalTestCase.load_test_data(None)`?